BraintreePayPal/PayPalDataCollector/Public/PPDataCollector.h

```diff
@@ -43,4 +43,14 @@
 */
 + (nonnull NSString *)collectPayPalDeviceData;
 
+/**
+Collects Device Info for PayPal.
+
+ This should be used when the user is paying with PayPal.
+ @param pairingID a pairing ID to associate with this clientMetadataID must be 10-32 chars long or null
+
+ @return a deviceInfo string that should be passed into server-side calls.
+ */
++ (nonnull NSString *)collectPayPalDeviceInfoWithClientMetadataID:(nullable NSString *)pairingID;
```

Comment:
Can I get more context on what a `pairingID` is?

Reply:
`pairingID` is another name for the `clientMetadataId`. It is the ID related for risk data.